src/TuDa.CIMS.Web/Components/ShoppingCartSubmitButton.razor.cs

```diff
@@ -0,0 +1,14 @@
+using MudBlazor;
+
+namespace TuDa.CIMS.Web.Components;
+
+public partial class ShoppingCartSubmitButton(IDialogService dialogService)
+{
+    /// <summary>
+    /// Opens the popup.
+    /// </summary>
+    private Task OpenDialogAsync(DialogOptions options)
+    {
+        return dialogService.ShowAsync<ShoppingCartSubmitPopup>("Custom Options Dialog", options);
+    }
```

Comment:
I meant directly insert the options as the parameter of `ShowAsync` and not adding a parameter to `OpenDialogAsync`